Makefile

```diff
@@ -40,9 +40,11 @@ gen-protos: protoc-gen-elixir
 	protoc -I src -I test/protobuf/protoc/proto --elixir_out=custom_field_options=true:test/protobuf/protoc/proto_gen --plugin=./protoc-gen-elixir test/protobuf/protoc/proto/extension.proto
 	protoc -I src -I test/protobuf/protoc/proto --elixir_out=custom_field_options=true:test/protobuf/protoc/proto_gen --plugin=./protoc-gen-elixir test/protobuf/protoc/proto/extension2.proto
 	protoc -I src -I test/protobuf/protoc/proto --elixir_out=custom_field_options=true:test/protobuf/protoc/proto_gen --plugin=./protoc-gen-elixir test/protobuf/protoc/proto/extension3.proto
+	protoc -I src -I test/protobuf/protoc/proto --elixir_out=custom_field_options=true:test/protobuf/protoc/proto_gen --plugin=./protoc-gen-elixir test/protobuf/protoc/proto/extension4.proto
 	protoc -I src -I test/protobuf/protoc/proto --elixir_out=custom_field_options=true:test/protobuf/protoc/proto_gen --plugin=./protoc-gen-elixir test/protobuf/protoc/proto/enum_options.proto
 	protoc -I src --elixir_out=lib --plugin=./protoc-gen-elixir elixirpb.proto
 	protoc -I src --elixir_out=lib --plugin=./protoc-gen-elixir brex_elixirpb.proto
 	protoc -I src  -I test/protobuf/protoc/proto/events --elixir_out=lib --plugin=./protoc-gen-elixir brex_events_extensions.proto
+	protoc -I src  -I test/protobuf/protoc/proto/demoscene --elixir_out=custom_field_options=true:lib --plugin=./protoc-gen-elixir demoscene_extensions.proto
```

Comment:
The `custom_field_options=true` part, I believe is needed to atomize the enums. We could do without that but it would be nice to be able to use `(brex.elixirpb.enum).atomize`

Reply:
Yeah, the syntax is weird here, but this looks correct to me